### ISSUE TITLE
Problem when redefining commands like \emph with \renewcommand

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#119]
am using the `Beamer` class and would like to redefine `\emph` to use `\bfseries` instead of `\itshape`. I have tried the following command:
````
\renewcommand<>{\emph}[1]{{\only#2{\bfseries}#1}}
````

but gets the following error
````
! LaTeX Error: Command \beamerx@\emph already defined. Or name \end... illegal, see p.192 of the manual.
````

I was told that this could be a possible bug. See

http://tex.stackexchange.com/questions/13726/how-to-redefine-the-emph-command-in-beamer

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/119
- Originally reported by: Ulrik Nielsen
- Originally created at:  2011-03-17

**Comment by josephwright** [comment]
_From Joseph Wright on 2011-03-17_

I'll take this one - should be sorted later today in the repo

**Comment by josephwright** [comment]
Fixed by 1567de1fe84860ed65d5c922e7e5fc6aab3f6d6c.
